src/test/old/testcrush.cc

```diff
@@ -44,7 +28,6 @@ Bucket *make_bucket(Crush& c, vector<int>& wid, int h, int& ndisks, int& nbucket
     UniformBucket *b = new UniformBucket(nbuckets--, 1, 0, 10, disks);
     b->make_primes(hash);  
     c.add_bucket(b);
-    //cout << h << " uniformbucket with " << wid[h] << " disks" << endl;
     return b;
   } else {
     // mixed
```

Comment:
@amitkumar50 

That is an informative comment in the `if {} else {}` that indicates UniformBucket or MixedBucket is to be made. It improves the readability of the code too, when someone reads it for the first time.